terraso_backend/tests/auth/test_views.py

```diff
@@ -46,6 +46,7 @@ def test_get_google_callback(client, access_tokens_google, respx_mock):
     assert response.status_code == 302
     assert response.cookies.get("atoken")
     assert response.cookies.get("rtoken")
+    assert response.cookies.get("sessionid")
```

Comment:
Question: Is the session ID a replacement to the tokens? I mean how will the clients use this session ID?

Reply:
The `sessionid` is not a replacement to the tokens. The web client won't be affected in any way by this new session. The `sessionid` token will only be used by the `/admin` views (and related views, like OAuth authorization).

It's important, though, that when the web client signs users out, it also calls to the `logout` endpoint to end the backend session. Otherwise, the user will sign out from the web client, but still be signed in to the admin pages. I created the issue so we can keep it tracked: https://github.com/techmatters/terraso-web-client/issues/281

Important: the admin pages will only be accessed by Terraso administrators, however the OAuth authorization page (that uses backend session) will be accessed by any user. When a non-staff (non-admin) user attempt to access the admin page, no resources are going to be showed.